src/openapi/streaming/subscription.js

```diff
@@ -35,6 +35,22 @@ const LOG_AREA = 'Subscription';
 
 // -- Local methods section --
 
+function getUrlTemplate(url, subscriptionData) {
+    if (!subscriptionData.Top) {
+        return url;
+    }
+
+    return url + '?$top={Top}';
+}
+
+function getTemplateArgs(subscriptionData) {
+    if (!subscriptionData.Top) {

```

Comment:
so I guess open api doesn't mind having the Top argument sent in its subscription data?

Reply:
I don't think it will affect anything but I will removed for clarity. Otherwise it can be misleading when evaluating requests.